src/components/Breadcrumb/Breadcrumb.component.js

```diff
@@ -1,46 +1,28 @@
-import React, { useState, useEffect } from 'react';
-import { useParams } from 'react-router-dom';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import propTypes from 'prop-types';
 
-import BreadcrumbContainer from './Breadcrumb.container';
+import Home from '../Home';
 
 import './style/breadcrumb.scss';
 
-const BreadcrumbComponent = () => {
-  const { path } = useParams();
-
-  const [showingRoute, setShowingRoute] = useState([]);
-  const [clickableFolders, setClickableFolders] = useState([]);
-
-  useEffect(() => {
-    const myRoute = path || '';
-    const cleanRoute = myRoute.replaceAll('%20', ' ');
-
-    const splitBreadCrumb = cleanRoute.split('/');
-    const cleanRoutes = splitBreadCrumb.filter((tempRoute) => tempRoute !== '');
-    const finalRoutes = [];
-
-    for (let i = 0; i < cleanRoutes.length; i += 1) {
-      let tempRoute = '';
-      for (let j = 0; j <= i; j += 1) {
-        tempRoute = `${tempRoute}/${cleanRoutes[j]}`;
+const BreadcrumbComponent = ({ breadcrumbs }) => (
+  <div className="directory-container">
+    <Home />
+    <div className="breadcrumb">
+      {
+        breadcrumbs.map(({ title, uri }) => (title ? (
+          <Link key={title} className="route-breadcrumb" to={`${uri}`}>
+            {`/${title}`}
+          </Link>
+        ) : undefined))
       }
-
-      finalRoutes.push(`${tempRoute}/`);
-    }
-
-    setClickableFolders(finalRoutes);
-    setShowingRoute(cleanRoutes);
-  }, [path]);
-
-  return (
-    <BreadcrumbContainer
-      clickableFolders={clickableFolders}
-      showingRoute={showingRoute}
-    />
-  );
-};
+    </div>
+  </div>
+);
 
 BreadcrumbComponent.propTypes = {
+  breadcrumbs: propTypes.arrayOf(Object).isRequired,
```

Comment:
Let's use `propTypes.arrayOf(propTypes.shape({})).isRequired`, and please fill the shape accordingly.